app/router.js

```diff
@@ -10,6 +10,7 @@ Router.map(function() {
   this.route('reminders', function() {
     this.route('reminder', { path: '/:reminder_id' });
```

Comment:
Oh, btw, this is chelle...we got this puppy going without wrapping edit.  We did what you said!  Please merge.